etc/Dockerfile.mod.pi

```diff
@@ -0,0 +1,50 @@
+ARG BASE_TAG
+ARG BASE_NAME
+ARG HUB_USER
+
+FROM ghcr.io/${HUB_USER}/${BASE_NAME}:${BASE_TAG}
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install software-properties-common to get apt-add-repository
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    software-properties-common
+
+# Add the Raspberry Pi repository
+RUN apt-add-repository 'deb http://archive.raspberrypi.org/debian/ bullseye main'
+
+# Import the repository's signing key
+RUN wget -qO - https://archive.raspberrypi.org/debian/raspberrypi.gpg.key | apt-key add -
+
+# Update the package list
+RUN apt-get -y update
+
+# Install GST dev files
+RUN apt-get -y install \
+    libcamera0 \
+    libgstreamer1.0-dev \
+    libgstreamer1.0-0 \
+    gstreamer1.0-x \
+    gstreamer1.0-tools \
+    gstreamer1.0-plugins-base \
+    gstreamer1.0-plugins-good \
+    gstreamer1.0-plugins-bad \
+    gstreamer1.0-plugins-ugly \
+    libgstreamer-plugins-base1.0-dev
+
+# Install GoogleTest
+RUN apt-get -y install libgtest-dev
+RUN cd /usr/src/gtest && \
+    cmake ./ && \
+    make
```

Comment:
We don't need to `make install` here and below, right?

Reply:
Unfortunately, there is no `make install` target until `v1.11`.

Here is the version that comes with debian bullseye apt sources:

```
Package: libgtest-dev
Version: 1.10.0.20201025-1.1
```